services/hsts/hsts.tester.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const { ServiceTester } = require('..')
+const label = 'hsts'
+
+const t = (module.exports = new ServiceTester({
+  id: 'hsts',
+  title: 'HSTS Preload',
+}))
```

Comment:
I think you should be able to use the shorthand function here to replace lines 3, and 6-9 
`const t = (module.exports = require('..').createServiceTester())`